root/static/scripts/edit/URLCleanup.js

```diff
@@ -1026,6 +1025,154 @@ const CLEANUPS = {
       return url;
     }
   },
+  a45cat: {
```

Comment:
You can also have keys begin with numbers if you quote them. (Assuming that's what the "a" is for, though it hardly matters, just thought I'd mention it.)